.werft/values.dev.gcp-storage.yaml

```diff
@@ -0,0 +1,50 @@
+components:
+  contentService:
+    remoteStorage:
+      kind: gcloud
+      backupTrail:
+        enabled: true
+        maxLength: 2
+      gcloud:
+        credentialsFileUpload: true
+        credentialsFile: /credentials/gitpod-dev-syncd-key.json
+        secretName: remote-storage-gcloud
+        projectId: gitpod-dev
+        region: europe-west1
+        tmpdir: /mnt/sync-tmp
+        parallelUpload: 6
+    volumes:
+      - name: gcloud-creds
+        secret:
+          secretName: remote-storage-gcloud
+    volumeMounts:
+    - mountPath: /credentials
+      name: gcloud-creds
+  wsManager:
+    volumes:
+    - name: gcloud-creds
+      secret:
+        secretName: remote-storage-gcloud
+    volumeMounts:
+    - mountPath: /credentials
+      name: gcloud-creds
+  wsDaemon:
+    volumes:
+    - name: gcloud-creds
+      secret:
+        secretName: remote-storage-gcloud
+    volumeMounts:
+    - mountPath: /credentials
+      name: gcloud-creds
+  server:
+    volumes:
+    - name: gcloud-creds
+      secret:
+        secretName: remote-storage-gcloud
+    serverContainer:
+      volumeMounts:
+      - mountPath: /storageKeySecret
+        name: gcloud-creds
+
+minio:
+  enabled: false
```

Comment:
This file looks like it should be in sync with
https://github.com/gitpod-io/gitpod/blob/main/install/gcp-terraform/modules/storage/templates/values.tpl